### PR TITLE
Sig ver param

### PR DESCRIPTION
**Pull request by stevenhorsman**
Add `enable_signature_verification` config support

### CODE REVIEW

File: src/agent/src/config.rs [hunk 1]
```diff
@@ -179,6 +182,7 @@ impl Default for AgentConfig {
             https_proxy: String::from(""),
             no_proxy: String::from(""),
             data_integrity: false,
+            enable_signature_verification: true,
```
Comment (fitzthum) [inline src/agent/src/config.rs:185]: It seems like https://github.com/kata-containers/tests/issues/5228 is saying that the default should be false?

Comment (stevenhorsman) [reply src/agent/src/config.rs:185]: Nope - the default is true (for security reasons), but we turn it off with the default kata-config for backwards compatibility

**Review by fidencio — APPROVED** [review]
lgtm, thanks @stevenhorsman!

@arronwy, mind to also take a look?

**Comment by stevenhorsman** [comment]
> @arronwy, mind to also take a look?

That would be great. I've heard that image-rs are considering making signature verification enabled by default soon, so I think this would allow that change to happen seamlessly for us, but let me know if you can spot something different.

File: src/agent/src/config.rs (hunk 1 above)
Comment (fitzthum) [reply src/agent/src/config.rs:185]: I wouldn't exactly call this security critical, but I guess it's not too big of a deal either way.

**Review by fitzthum — APPROVED** [review]

**Review by arronwy — APPROVED** [review]
Yes, Thanks @stevenhorsman , for security consideration `image-rs` will enable it by default, with this PR, user can easy to switch on/off this feature if needed.

**Pull request merged**